test/test_models.py

```diff
@@ -69,6 +70,13 @@ def get_available_video_models():
 
 
 class ModelTester(TestCase):
+
+    def check_fx_compatible(self, model, inputs):
+        model_fx = torch.fx.symbolic_trace(model)
+        out = model(inputs)
+        out_fx = model_fx(inputs)
+        self.assertTrue(out.equal(out_fx))
```

Comment:
do the models output classes? otherwise we may need a less strict assertion

Reply:
They output the logits, so there is no max operation performed here.

Reply:
Should we use `torch.allclose` then?

Reply:
How is the execution time increased after introducing this? If we have a significant slowdown, it might be worth not reestimating `model(input)` but rather pass it from there one of the other places where we calculate it.

Reply:
> Should we use torch.allclose then?

Sure, we can change it to use `torch.allclose`.

> How much is the execution time increased after introducing this

Good question. I've considered this when writing the tests but went with the same approach that the torchscript tests do to recompute the output.

By comparing the runtimes from this diff and https://github.com/pytorch/vision/pull/3663 (which are based off on the same base), the runtimes don't seem to change significantly

Before https://app.circleci.com/pipelines/github/pytorch/vision/7409/workflows/a7e5719e-7054-4019-bfe9-7c4a5e840900/jobs/511741:
```
9.50s call     test/test_models.py::ModelTester::test_densenet169_cpu
```

this PR https://app.circleci.com/pipelines/github/pytorch/vision/7408/workflows/b9b610f9-6453-4d8e-ac1b-7b7140ebfe26/jobs/511600
```
9.55s call     test/test_models.py::ModelTester::test_densenet161_cpu
```

and the total runtimes don't really seem affected, so I'd say we can keep it like this.

Reply:
> and the total runtimes don't really seem affected, so I'd say we can keep it like this.

Yeap agreed. Since there is no visible slowdown, no point introducing more convoluted code.